test/ModalSpec.js

```diff
@@ -220,6 +220,24 @@ describe('Modal', function () {
     });
   });
 
```

Comment:
Good point. It will increase [coveralls.io](https://coveralls.io/github/react-bootstrap/react-bootstrap), which has been suffered a little last time :smile: 
[![Coverage Status](https://coveralls.io/repos/react-bootstrap/react-bootstrap/badge.svg?branch=master&service=github)](https://coveralls.io/github/react-bootstrap/react-bootstrap?branch=master)